packages/app-builder/src/utils/environment.ts

```diff
@@ -106,7 +107,11 @@ export function getServerEnv<K extends keyof ServerEnvVarName>(
 export function getClientEnvVars() {
   return {
     ENV: getServerEnv('ENV'),
-    AUTH_EMULATOR_HOST: getServerEnv('FIREBASE_AUTH_EMULATOR_HOST'),
+    FIREBASE_AUTH_EMULATOR_HOST: getServerEnv(
+      'FIREBASE_AUTH_EMULATOR_HOST',
+    )?.startsWith('http://')
+      ? getServerEnv('FIREBASE_AUTH_EMULATOR_HOST')
+      : 'http://' + getServerEnv('FIREBASE_AUTH_EMULATOR_HOST'),
```

Comment:
Why do you need this ?

Reply:
ok good question. The reason was that currently we have `http://localhost:9099` as the value for the env variable in the example env (which is not really a hostname). And it was confusing in the docker compose because the backend expects the hostname without http.
But in fact in the doc they put it without the protocol https://firebase.google.com/docs/emulator-suite/connect_auth#admin_sdks so I guess we can just keep it as `localhost:9099` in our example en after all

Reply:
Yes, I think we can harmonize this and make the change once for all (IIRC, you already raised this issue for other env vars).